misc/services/mobile-webui/mobile-webui-frontend/src/containers/activities/picking/PickAltStepButton.jsx

```diff
@@ -0,0 +1,104 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { push } from 'connected-react-router';
+import counterpart from 'counterpart';
+
+import { pushHeaderEntry } from '../../../actions/HeaderActions';
+import Indicator from '../../../components/Indicator';
+import * as CompleteStatus from '../../../constants/CompleteStatus';
+
+class PickAltStepButton extends PureComponent {
```

Comment:
IMHO this shall be the same as a regular picking step button.
The only diff is that u have to pass the `alternativeId` around.